docs/en/week06/06-3.md

```diff
@@ -221,38 +211,77 @@ How do we turn off the output? Let’s say we have a purple internal representat
+<b>Figure 22:</b>LSTM Architecture - Write Memory
+</center>
 
 ## Notebook Examples
 
 ### Sequence Classification
 
-The goal is to classify sequences. Elements and targets are represented locally (input vectors with only one non-zero bit). The sequence starts with an B, ends with a E (the “trigger symbol”), and otherwise consists of randomly chosen symbols from the set {a, b, c, d} except for two elements at positions t1 and t2 that are either X or Y. For the DifficultyLevel.HARD case, the sequence length is randomly chosen between 100 and 110, t1 is randomly chosen between 10 and 20, and t2 is randomly chosen between 50 and 60. There are 4 sequence classes Q, R, S, and U, which depend on the temporal order of X and Y. The rules are: X, X -> Q; X, Y -> R; Y, X -> S; Y, Y -> U.
+The goal is to classify sequences. Elements and targets are represented locally (input vectors with only one non-zero bit). The sequence starts with an $B$, ends with a $E$ (the “trigger symbol”), and otherwise consists of randomly chosen symbols from the set $\{a, b, c, d\}$ except for two elements at positions $t1$ and $t2$ that are either $X$ or $Y$. For the DifficultyLevel.HARD case, the sequence length is randomly chosen between 100 and 110, $t1$ is randomly chosen between 10 and 20, and $t2$ is randomly chosen between 50 and 60. There are 4 sequence classes $Q, R, S$, and $U$, which depend on the temporal order of $X$ and $Y$. The rules are: `X, X -> Q`; `X, Y -> R`; `Y, X -> S`; `Y, Y -> U`.
 
 1). Dataset Exploration
 
-The return type from a data generator is a tuple with length 2. The first item in the tuple is the batch of sequences with shape (32, 9, 8). This is the data going to be fed into the network. There are eight different symbols in each row (X, Y, a,b, c, d, B, E). Each row is a one-hot vector. Sequence of rows represents sequence of symbols. The first all-zero row is padding. We use padding when the length of sequence is shorter than the maximum length in the batch.  The second item in the tuple is the corresponding batch of class labels with shape (32, 4), since we have 4 classes (Q, R, S, and U). The first sequence is: BbXcXcbE. Then its decoded class label is $[1, 0, 0, 0]$, corresponding to Q.
+The return type from a data generator is a tuple with length 2. The first item in the tuple is the batch of sequences with shape $(32, 9, 8)$. This is the data going to be fed into the network. There are eight different symbols in each row $(X, Y, a,b, c, d, B, E)$. Each row is a one-hot vector. Sequence of rows represents sequence of symbols. The first all-zero row is padding. We use padding when the length of sequence is shorter than the maximum length in the batch.  The second item in the tuple is the corresponding batch of class labels with shape $(32, 4)$, since we have 4 classes $(Q, R, S, and U)$. The first sequence is: `BbXcXcbE`. Then its decoded class label is $[1, 0, 0, 0]$, corresponding to $Q$.
```

Comment:
```suggestion
The return type from a data generator is a tuple with length 2. The first item in the tuple is the batch of sequences with shape $(32, 9, 8)$. This is the data going to be fed into the network. There are eight different symbols in each row (`X`, `Y`, `a`, `b`, `c`, `d`, `B`, `E`). Each row is a one-hot vector. Sequence of rows represents sequence of symbols. The first all-zero row is padding. We use padding when the length of sequence is shorter than the maximum length in the batch.  The second item in the tuple is the corresponding batch of class labels with shape $(32, 4)$, since we have 4 classes (`Q`, `R`, `S`, and `U`). The first sequence is: `BbXcXcbE`. Then its decoded class label is $[1, 0, 0, 0]$, corresponding to `Q`.
```